device-types/Dell/PowerEdge-R760xd2.yaml

```diff
@@ -0,0 +1,87 @@
+---
+manufacturer: Dell
+model: PowerEdge R760xd2
+slug: dell-poweredge-r760xd2
+u_height: 2
+is_full_depth: true
+front_image: true
+rear_image: true
+airflow: front-to-rear
+weight: 46.3
+weight_unit: kg
+comments: |
+  [PowerEdge R760xd2 spec sheet](https://www.delltechnologies.com/asset/fr-fr/products/servers/technical-support/poweredge-r760xd2-spec-sheet.pdf)
+  [PowerEdge R760xd2 tech guide](https://www.delltechnologies.com/asset/en-us/products/servers/technical-support/poweredge-r760xd2-technical-guide.pdf)
+rear-ports:
+  - name: Rear USB 2
+    type: other
+  - name: Rear USB 3
+    type: other
+  - name: iDrac
+    type: other
+front-ports:
+  - name: Front USB 2
+    type: other
+    rear_port: Rear USB 2
```

Comment:
These are not suppose to be modelled here. Regular USB ports are not suppose to be modelled in NetBox at the moment unless they are console ports. Please remove them.